packages/plugins/rubygem-smart_proxy_request_forwarder/rubygem-smart_proxy_request_forwarder.spec

```
@@ -0,0 +1,87 @@
+# template: smart_proxy_plugin
+%global gem_name smart_proxy_request_forwarder
+%global plugin_name request_forwarder
+
+%global foreman_proxy_min_version 1.24
```

Does this indicate that the plugin works with foreman-proxy 1.24? I don't think we should make this promise.

Maybe:

```suggestion
%global foreman_proxy_min_version 3.11
```

---

@evgeni Any idea why it is using this foreman_proxy_min_version? Is it the default one? I don't think that we set one in the gemspec file of the plugin itself.

---

yeah, 1.24 is hard-coded in the template: https://github.com/theforeman/foreman-packaging/blob/rpm/develop/gem2rpm/smart_proxy_plugin.spec.erb#L15

set whatever you think is appropriate